### ISSUE TITLE
`aboveBarData` relative cutoff

### ISSUE THREAD

**Issue by zkrige** [imaNNeoFighT/fl_chart#769]
We want to make a graph with `aboveBarData` that cuts off like this.
<img width="672" alt="image" src="https://user-images.githubusercontent.com/968412/133257454-903f2f5a-6815-4211-b370-ef53b38407b1.png">

Currently the `cutOffY` is absolute, so there isn't an easy way to do this.

Could you add a flag that `cutOffY` is relative instead of absolute. If relative and is set to "1", then it will cutoff 1 above the spot, instead of cutting off at 1 on the graph

**Comment by imaNNeoFighT** [comment]
It's a little hard to implement. (It seems simple in your use case. But I'm thinking of curved line charts...)

**Comment by imaNNeoFighT** [comment]
BTW Let us think about it.

**Comment by zkrige** [comment]
Yes I appreciate there's probably complexities beyond my use case.

thank you
